CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# 0.2.0
+
+Bottlerocket Update Operator (Brupop) 0.2.0 is a complete overhaul and rewrite of the update operator. It will, by default, continue to rely on Bottlerocket’s client-side update API to determine when to perform an update on any given node — foregoing any complex deployment velocity controls, and instead relying on the wave system built-in to update Bottlerocket. Compared to Brupop 0.1.0, Brupop 0.2.0 not only improves performance but also increases observability and scopes down permissions required by the update operator agent.
```

Comment:
nit. Brupop 0.2.0 not only improves performance, but also increases observability while scoping down permissions required by the update operator agent.